yb-voyager/cmd/common.go

```diff
@@ -669,3 +720,61 @@ func renameTableIfRequired(table string) string {
 	}
 	return table
 }
+
+
+func getExportedSnapshotRowsMap(tableList []string, exportSnapshotStatus *ExportSnapshotStatus) (map[string]int64, map[string]string, error) {
+	snapshotRowsMap := make(map[string]int64)
+	snapshotStatusMap := make(map[string]string)
+	for _, table := range tableList {
+		renamedTable := renameTableIfRequired(table)
+		snapshotStatusMap[renamedTable] = exportSnapshotStatus.Tables[table].Status
```

Comment:
What do we display here as part of "status"? It'd be technically wrong I guess because we're simply taking the last leaf table's status to be the root table's status?

Reply:
Yeah, I had fixed it properly in import data status. Missed here will do.